vcd/structure.go

```diff
@@ -85,6 +85,9 @@ func getProtocol(protocol types.FirewallRuleProtocols) string {
 	if protocol.TCP {
 		return "tcp"
 	}
+	if protocol.TCP && protocol.UDP {
+		return "tcp&udp"
```

Comment:
I'm a bit surprised we have lived without this so far :) But at the same time it looks like we're _not_ using this function for NSX-V FW rule?

CC: @Didainius

Reply:
it was used only in vcd_firewall_rules